python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -530,25 +529,34 @@ def is_generic_test_on_attached_node_from_dbt_resource_props(
 def default_asset_check_fn(
     asset_key: AssetKey,
     unique_id: str,
-    dagster_dbt_translator_settings: "DagsterDbtTranslatorSettings",
+    dagster_dbt_translator: "DagsterDbtTranslator",
     dbt_resource_props: Mapping[str, Any],
+    dbt_nodes: Mapping[str, Any],
+    parent_ids: List[str],
 ) -> Optional[AssetCheckSpec]:
```

Comment:
Just use `manifest`, so we don't have to pass in two arguments. Refactor `default_asset_check_fn` to the following arguments:

- `manifest`,
- `dagster_dbt_translator`,
- `asset_key`,
- `unique_id`
- `test_unique_id`

Then all the necessary indexing into `manifest` can be done in the body of `default_asset_check_fn`.